src/configuration/locales/fr/translations.json

```diff
@@ -1,3 +1,33 @@
 {
-  "Take the survey!": "Réponder au questionnaire!"
+  "...Loading": "...Chargement",
+  "Beekeeper": "Apiculteur",
+  "Beginner": "Débutant",
+  "Billed: {{value}}": "Payant: {{value}}",
+  "Description {{description}}:": "Description: {{description}}",
+  "Experience: {{experience}}": "Expérience: {{experience}}",
+  "Expert": "Expert",
+  "FreeSpace": "Espace disponible",
+  "Garden": "Jardin",
+  "Happyculteur © Copyright 2019. All rights reserved.": "Happyculteur © Copyrights 2019. Tous droits réservés.",
+  "Happyculteur partner!": "Partenaire Happyculteur!",
+  "Individual": "Particulier",
+  "Install": "Installer des ruches",
+  "Interests:": "Intérêts:",
+  "Legal notice": "Mentions légales",
+  "Learn": "Apprendre",
+  "Meadow": "Prairie",
+  "Medium": "Moyen",
+  "Public:": "Public:",
+  "Roof": "Toit",
+  "Special thanks": "Remerciements tout particuliers",
+  "Scroll to load more": "Faire défiler pour charger",
+  "Sell": "Vendre",
+  "Since: {{seniority}} years": "Depuis: {{seniority}} années",
+  "Size: {{size}}": "Taille: {{size}}",
+  "Space": "Espace",
+  "Take the survey!": "Répondre au questionnaire!",
```

Comment:
`Répondre au questionnaire !`
There is a space between the word and the exclamation punctuation in French